client/src/sections/resources/Cartography.tsx

```diff
@@ -0,0 +1,168 @@
+            ")"
+          : " @ " + this.zone)
+      );
+    } else {
+      return " • " + this.monster + " @ " + this.zone;
+    }
+  }
+}
+
+/**
+ * Summarizes # of maps remaining & recommends usage ideas
+ * @returns A tile describing the Backup Camera
+ */
+
+const Cartography = () => {
+  const _mapUses = 3 - useGet("_monstersMapped", 0);
+  const userLevel = useMyLevel() ?? 0;
+
+  // This next portion tabulates the number of turns until GROPs are unlocked. You need these quest prefs to know how many hippies a single war turn advances the war.
+  const lighthouseQuest = +(
+    useGet("sidequestLighthouseCompleted") === "fratboy"
+  );
+  const arenaQuest = +(useGet("sidequestArenaCompleted") === "fratboy");
+  const junkyardQuest = +(useGet("sidequestJunkyardCompleted") === "fratboy");
+  const orchardQuest = +(useGet("sidequestOrchardCompleted") === "fratboy");
+  const nunsQuest = +(useGet("sidequestNunsCompleted") === "fratboy");
+  const hippiesPerFight = Math.pow(
+    2,
+    nunsQuest + orchardQuest + junkyardQuest + arenaQuest + lighthouseQuest
+  );
+
+  // Battlefield spading from Aen shows that GROPs appear @ 400 hippy kills, but not before.
+  const hippiesKilled = useGet("hippiesDefeated");
+  const turnsToGROPs = Math.ceil(
+    Math.max(401 - hippiesKilled, 0) / hippiesPerFight
+  );
+
+  // General access booleans to pass into my Map Target list
+  const gropReqs = !(useGet("warProgress") === "finished");
+  const healerReqs = !useHave($item`amulet of extreme plot significance`);
+  const hitsReq = useGet("questL10Garbage") in ["step10", "finished"];
+
+  // Properties referenced by multiple mapTargets
+  const zeppProgress = useGet("questL11Ron");
+
+  // This lists out possible map targets. Currently just three guys.
+  const allMapTargets = [
+    new mapTarget(
+      "Green Ops Soldier",
+      "The Battlefield",
+      12,
+      gropReqs,
+      turnsToGROPs
+    ),
+    new mapTarget("Quiet Healer", "Penultimate Airship", 10, healerReqs),
+    new mapTarget("Lobsterfrogman", "Sonofa Beach", 12, !junkyardQuest),
+    new mapTarget("Astronomer", "The Hole in the Sky", 10, hitsReq),
+    new mapTarget(
+      "Red Butler",
+      "The Red Zeppelin",
+      11,
+      zeppProgress in ["step2", "step3"]
+    ),
+    new mapTarget(
+      "Lynyrd Skinner",
+      "Mob of Zeppelin Protestors",
+      11,
+      zeppProgress in ["started", "step1"]
+    ),
+    new mapTarget("Forest Spirit", "Outskirts of Camp Logging Camp", 4),
+  ];
+
+  // Once I have more map targets here, I'll pull in the recc code from Camel.
+  const recommendations = allMapTargets;
+
+  return (
+    <Tile
+      header="Cartography Compendium"
+      imageUrl="/images/itemimages/Cccbook.gif"
+      hide={!useHave($skill`Comprehensive Cartography`) || _mapUses === 0}
+    >
+      <Line>You have {_mapUses} maps remaining. Map the monster ideas:</Line>
+      {recommendations[0].accessible(userLevel) && (
```

Comment:
You can do `recommendations.slice(0, 3).map(xxx)` instead of repeating yourself here.